lib/modules/dosomething/dosomething_signup/dosomething_signup.query.inc

```diff
@@ -54,8 +54,10 @@ function dosomething_signup_get_signups_query($params, $tally = FALSE) {
 function dosomething_signup_build_signups_query($params = []) {
   $query = db_select('dosomething_signup', 's');
   $query->leftJoin('dosomething_reportback', 'rb', 's.uid = rb.uid and s.nid = rb.nid and s.run_nid = rb.run_nid');
+  $query->leftJoin('field_data_field_northstar_id', 'ns', 'ns.entity_id = s.uid');
```

Comment:
Just as a heads up, this field is not guaranteed to be populated for every account. We're currently running a script that's setting it on each user, but it won't be done for a while yet.